addons/website/static/src/js/editor/snippets.options.js

```diff
@@ -192,6 +193,16 @@ const FontFamilyPickerUserValueWidget = SelectUserValueWidget.extend({
                         }
 
                         const font = m[1].replace(/\+/g, ' ');
+                        // Do not add the font if it has already been added.
+                        // TODO in V16 : Add it if one is imported on the
+                        // server and the other is not.
```

Comment:
Nitpicking (adding it as there is a real change to make anyway with the .pot file):
Use the whole space up to 80, don't return before if you still have room for the next word
```suggestion
                        // TODO in V16: Add it if one is imported on the server
                        // and the other is not.
```